paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/Spec.kt

```diff
@@ -21,24 +20,33 @@ data class LayoutSpec(val elements: List<FormElementSpec>) {
         get() = elements.filterIsInstance<SectionSpec>().map { it.field }
 }
 
+data class IdentifierSpec(val value: String)
+
 /**
  * This is used to define each section in the visual form layout
  */
 
```

Comment:
extra line

Reply:
ktlint formatter will be in an upcoming review.